contexts/compound-and-earn-context.js

```diff
@@ -465,9 +482,26 @@ export function CompoundAndEarnProvider({ children }) {
 
       if (item.kind === 'Snowglobe') {
         const snowglobeContract = new ethers.Contract(item.address, SNOWGLOBE_ABI, library.getSigner());
+        
+        var snowglobeBalance = ethers.BigNumber.from("0");
+
+        var currentDepth = 0;
+        const retryBalance = async () => {
+          snowglobeBalance = await snowglobeContract.balanceOf(account);
+          if(snowglobeBalance.gt("0x0")){
+            return true;
+          }else{
+            if(MAX_RETRIES > currentDepth){
+              currentDepth++;
+              await retryBalance();
+            }
+          }
+        }
+
         //await 2 seconds for our node to sync
         await delay(2000);
```

Comment:
is this delay still necessary?

Reply:
without the delay I assume that the first try will mostly fail